src/pages/Homepage/Homepage.tsx

```diff
@@ -51,6 +77,35 @@ export const Homepage = () => {
           classList={styles.aboutSection}
           color="white"
         />
+        <div className={styles.customersContainer}>
+          <div className={styles.customers}>
+            {data.map(({ name, position, clientPhoto, company }, index) => (
+              <ThumbnailCard
+                name={name}
+                position={position}
+                clientPhoto={clientPhoto}
+                company={company}
+                key={index}
+                isSelected={selectedItem?.company === company}
+                handleClick={() =>
+                  handleCardSelect({ name, position, clientPhoto, company })
+                }
+              />
+            ))}
+          </div>
+          <div className={styles.cardDetailsContainer}>
+            {cardDetailsData && (
+              <CardDetails
```

Comment:
we dont want the div wrapper beign rendered if `cardDetailsData` is undefined

Reply:
I meant if cardDetailsData is falsy we will render an empty div. -- we don't want that so replace it for this

```tsx
{ cardDetailsData ? (
    <div className={styles.cardDetailsContainer}>
      <CardDetails ... />
    </div>
  ) : null
}
```